lib/Reader/readerir.cpp

```diff
@@ -3746,7 +3763,9 @@ IRNode *GenIR::genArrayElemAddress(IRNode *Array, IRNode *Index,
                                    Type *ElementTy) {
```

Comment:
I'm surprised this method (`genArrayElemAddress`) didn't need a call to `mdArrayRefAddr` for the MD case.